data/json/overmap_specials.json

```diff
@@ -1766,5 +1766,63 @@
         "city_sizes" : [4, 12],
         "rotate" : false,
         "occurrences" : [1, 4]
-    }
+    },
+  {
+    "type": "overmap_special",
+    "id": "State Park",
+    "overmaps": [
+      { "point": [ 1, -1, 0 ], "overmap": "road_end_north" },
+      { "point": [ 0, 0, 0 ], "overmap": "state_park_0_0_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "state_park_1_0_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "state_park_0_1_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "state_park_1_1_north" }
+    ],
+    "connections": [
+      { "point": [ 1, -1, 0 ], "terrain": "road" }
+    ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ -1, 3 ],
+    "city_sizes": [ 4, 12 ],
+    "occurrences": [ 1, 5 ]
+  },
+  {
+    "id": "Fishing Pond",
+    "type": "overmap_special",
+    "overmaps": [
+      { "point": [ 1, -1, 0 ], "overmap": "road_end_north" },
+      { "point": [ 0, 0, 0 ], "overmap": "fishing_pond_0_0_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "fishing_pond_1_0_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "fishing_pond_0_1_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "fishing_pond_1_1_north" }
+    ],
+    "connections": [
+      { "point": [ 1, -1, 0 ], "terrain": "road" }
+    ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ -1, 3 ],
+    "city_sizes": [ 4, 12 ],
+    "occurrences": [ 1, 5 ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Derelict Property",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "derelict_property_north" }
+    ],
+    "locations": [ "wilderness" ],
+    "city_distance": [ 1, 10 ],
+    "city_sizes": [ 1, 12 ],
+    "occurrences": [ 4, 10 ]
```

Comment:
All minimal occurrences are going against the "budget". They should be kept low until we have a fix for overmap special limit.